arangod/Sharding/ShardingInfo.h

```diff
@@ -114,11 +115,11 @@ class ShardingInfo {
   size_t _numberOfShards;
 
   // @brief replication factor (1 = no replication, 0 = smart edge collection)
-  size_t _replicationFactor;
+  std::atomic<size_t> _replicationFactor;
```

Comment:
Why does this need to be atomic? As far as I know it is only ever written to in the beginning and then never changed? Maybe add a comment to explain this.

Reply:
Added a comment.